apis/python/src/tiledbsoma/query_condition.py

```diff
@@ -91,20 +91,20 @@ class QueryCondition:
 
         ``val ::= <num> | <str> | val(val)``
 
-    **Example:**
-
-    >>> with tiledb.open(uri, mode="r") as A:
-    >>>     # Select cells where the attribute values for `foo` are less than 5
-    >>>     # and `bar` equal to string "asdf".
-    >>>     # Note precedence is equivalent to:
-    >>>     # tiledbsoma.QueryCondition("foo > 5 or ('asdf' == attr('b a r') and baz <= val(1.0))")
-    >>>     A.query(cond="foo > 5 or 'asdf' == attr('b a r') and baz <= val(1.0)")
-    >>>
-    >>>     # Select cells where the attribute values for `foo` are equal to
-    >>>     # 1, 2, or 3.
-    >>>     # Note this is equivalent to:
-    >>>     # tiledbsoma.QueryCondition("foo == 1 or foo == 2 or foo == 3")
-    >>>     A.query(cond="foo in [1, 2, 3]")
+    Example::
+
+        with tiledb.open(uri, mode="r") as A:
```

Comment:
It is quite confusing (to me!) to have a TileDB-Py example in the middle of the `tiledbsoma` package. It suggest that the TileDB-Py syntax/classes are workable, eg., I can hand a QueryCondition instance to a `tiledbsoma` class, or something?

Options:
* [best] rewrite to use the SOMA API
* delete

Reply:
Fixed, I believe. Please double-check my syntax here.

Reply:
Fixed, I believe. Please double-check my syntax here.

Reply:
love the good website that sometimes posts comments twice